okapi-core/src/main/raml/RoutingEntry.json

```diff
@@ -81,7 +81,7 @@
       }
     },
     "permissionsRequiredTenant": {
-      "description": "Required permissions when tenant matches pathPattern",
+      "description": "Required permissions that replace permissionsRequired when the user is authenticated and either the tenant matches each {tenantId} in pathPattern or pathPattern doesn't contain {tenantId}",
```

Comment:
Just for the record, this field `permissionsRequiredTenant` and the new description look confusing (to me). It is also debatable whether this is a **breaking** change or not. Luckily only Okapi uses this at the moment.